docs/content/internals/Release.md

```diff
@@ -18,42 +18,41 @@ If it is `Failed` you could release from previous `Succeed` commits or fix probl
 This branch must be persistent. It is used for automation.
 1. Manually run [Integration build](http://links.k.avito.ru/ZA) on the `release branch`.
 1. Manually run [Github publish configuration](http://links.k.avito.ru/releaseAvitoTools) on the `release branch`.
+1. Verify and close staging repo on [nexus](https://oss.sonatype.org/#stagingRepositories). See `Maven central publishing reference article` below
```

Comment:
This is a long article with lots of details. 
As next steps, It'd good to extract exact steps in a "How to" manner.